deploy/metrics-server-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: metrics-server
+  namespace: kube-system
+  labels:
+    kubernetes.io/name: "Metrics-server"
```

Comment:
The other services start with capital letter:
https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/cluster-monitoring/influxdb